release-blog/2025/2025-03-19.0.mdx

```diff
@@ -0,0 +1,167 @@
+| base | land_use | data_changed | 663489 |
+| base | land_use | removed | 41562 |
+| base | land_use | unchanged | 47065761 |
+| base | water | added | 603639 |
+| base | water | data_changed | 58313 |
+| base | water | removed | 31885 |
+| base | water | unchanged | 54706153 |
+
+
+### Buildings
+
+- added ~13M new buildings from [Instituto Geográfico Nacional, Spain](https://www.ign.es)
+
+| theme | type | change type | feature count |
+| -------- | -------- | ------- | ------- |
+| buildings | building | added | 20895759 |
+| buildings | building | data_changed | 10104299 |
+| buildings | building | removed | 9188886 |
+| buildings | building | unchanged | 2598324355 |
+| buildings | building_part | added | 32568 |
+| buildings | building_part | data_changed | 56729 |
+| buildings | building_part | removed | 3101 |
+| buildings | building_part | unchanged | 3136373 |
+
+
+### Divisions
+
+- corrected format of OSM source ids, which now include the @version format, as seen in other themes
+- removed an incorrect Argentina perspective field value
+- the `local_type` value is set on regions in the US and India
+- all division boundary GERS IDs have changed since the February release
+
+| theme | type | change type | feature count |
+| -------- | -------- | ------- | ------- |
+| divisions | division | added | 40462 |
+| divisions | division | data_changed | 621942 |
+| divisions | division | removed | 33684 |
+| divisions | division | unchanged | 3699466 |
+| divisions | division_area | added | 26620 |
+| divisions | division_area | data_changed | 46748 |
+| divisions | division_area | removed | 23266 |
+| divisions | division_area | unchanged | 945111 |
+| divisions | division_boundary | added | 86684 |
+| divisions | division_boundary | data_changed | 2 |
+| divisions | division_boundary | removed | 86756 |
+| divisions | division_boundary | unchanged | 27 |
+
+
+### Places
+
+- minor data fixes and updates
+
+| theme | type | change type | feature count |
+| -------- | -------- | ------- | ------- |
+| places | place | added | 17185352 |
+| places | place | data_changed | 16596848 |
+| places | place | removed | 12857805 |
+| places | place | unchanged | 32815899 |
+
+### Transportation
+
+- added an [is_indoor road flag](https://github.com/OvertureMaps/schema/pull/327#event-16716730346) to the schema
+- re-included 88,005 segments flagged as ‘indoor’, which had previously been filtered out
+
+| theme | type | change type | feature count |
+| -------- | -------- | ------- | ------- |
+| transportation | connector | added | 1774872 |
+| transportation | connector | data_changed | 815586 |
+| transportation | connector | removed | 311378 |
+| transportation | connector | unchanged | 370644827 |
+| transportation | segment | added | 1698190 |
+| transportation | segment | data_changed | 4171164 |
+| transportation | segment | removed | 635288 |
+| transportation | segment | unchanged | 314306310 |
+
+  
+## Data changelog
```

Comment:
I notice this name has changed from Global Entity Reference System (GERS) changelog in the previous releases. Just flagging to confirm that is intended.

Reply:
Realizing this was probably intentional - @danabauer should this (and references) be "Data changelog" or "GERS changelog"